setup.py

```diff
@@ -53,66 +54,85 @@ def run(self):
                 try:            os.remove(g)
                 except OSError: pass
 
+# Platform information
+python2 = (sys.version_info[0] == 2)
+windows = sys.platform.startswith("win")
 
 # If cython is present, we'll compile
 # the pyx files from scratch. Otherwise,
 # we'll compile the pre-generated c
 # files (which are assumed to be present).
-
-# We only need numpy to
-# compile the test modules
 have_cython = True
-have_numpy  = True
-
+build_tests  = True
 
 try:    from Cython.Build import cythonize
 except: have_cython = False
 
+# We need numpy to compile the test modules
 try:    import numpy as np
-except: have_numpy = False
+except: build_tests = False
 
+include_dirs = ['indexed_gzip']
+lib_dirs = []
+libs = []
+extra_compile_args = []
 
 # If numpy is present, we need
 # to include the headers
-include_dirs = ['indexed_gzip']
-if have_numpy:
+if build_tests:
     include_dirs.append(np.get_include())
 
+if windows:
```

Comment:
I think it would be fine to just assume the presence of e.g. a `ZLIB_HOME` environment variable, and put the onus on the user to tell us where zlib is installed.